docroot/themes/custom/yukonca_glider/yukonca_glider.theme

```diff
@@ -344,3 +344,20 @@ function yukonca_glider_preprocess_file_link(&$variables) {
   $variables['link']['#attributes']['target'] = '_blank';
   $variables['filename'] = $filename;
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ *
+ * Limit the text length of the last breadcrumb item.
+ */
+function yukonca_glider_preprocess_breadcrumb(&$variables) {
+  $max_length = 100;
+  $index = array_key_last($variables['breadcrumb']);
+  $title = $variables['breadcrumb'][$index]['text'];
+  if (strlen($title) > $max_length) {
+    $variables['breadcrumb'][$index]['text'] = substr($title, 0, $max_length) . '...';
+  }
+  // Add the cache tag to update the breadcrumb if the node title changes.
+  $nid = \Drupal::routeMatch()->getRawParameter('node');
```

Comment:
Shouldn't we add a condition to add this tag only on node pages?
Because on the views pages, this code doesn't make sense